cpp/oneapi/dal/algo/knn/test/batch.cpp

```diff
@@ -39,92 +39,63 @@ namespace la = te::linalg;
 template <typename TestType>
 class knn_batch_test : public te::float_algo_fixture<std::tuple_element_t<0, TestType>> {
 public:
-    using Float = std::tuple_element_t<0, TestType>;
-    using Method = std::tuple_element_t<1, TestType>;
+    using float_t = std::tuple_element_t<0, TestType>;
+    using method_t = std::tuple_element_t<1, TestType>;
```

Comment:
Is this consistent with other tests?

Reply:
Yes. We have an agreement that only types in template parameters should be written in CamelCase. In other case it should be done in snake_case